src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/PercentToLocalizedStringTransformerTest.php

```diff
@@ -118,4 +118,115 @@ public function testReverseTransformExpectsString()
 
         $transformer->reverseTransform(1);
     }
+
+    public function testDecimalSeparatorMayBeDotIfGroupingSeparatorIsNotDot()
+    {
+        \Locale::setDefault('fr');
```

Comment:
tests are failing because this method is not always available, can you fix it @aaa2000 ?